server/src/uuid/model/entity/mod.rs

```diff
@@ -990,6 +995,64 @@ impl Entity {
     }
 }
 
+impl Entity {
+    pub async fn deleted_entities<'a, E>(
+        payload: &deleted_entities_query::Payload,
+        executor: E,
+    ) -> Result<Vec<DeletedEntity>, sqlx::Error>
+    where
+        E: Executor<'a>,
+    {
+        let date_database: Option<DateTime>;
+        let test = payload.after.as_ref();
+        match test {
+            Some(date) => {
+                let date_chrono = chrono::DateTime::parse_from_rfc3339(&(date))
+                    .unwrap()
```

Comment:
@hugotiburtino  Shall I catch the error (wrong format) here?

Reply:
I was thinking about that, see https://github.com/serlo/serlo.org-database-layer/pull/240#discussion_r867970518